er.sh

```diff
@@ -479,8 +479,8 @@ F_SIGPATCH(){
     ERR=$?
     echo "[$FUNCNAME] Signature spoofing patching ended with $ERR"
     echo "[$FUNCNAME] adding signature spoof controller"
-    cp $PDIR/*-packages-apps-Settings-src-com-android-settings-development-SpoofSignaturePreferenceController.java ${SRC_TOP}/packages/apps/Settings/src/com/android/settings/development/SpoofSignaturePreferenceController.java || exit 3
-    cp $PDIR/*-packages-apps-Settings-src-com-android-settings-development-SpoofSignatureInfo.java ${SRC_TOP}/packages/apps/Settings/src/com/android/settings/development/SpoofSignatureInfo.java || exit 3
+    [ -f $PDIR/*-packages-apps-Settings-src-com-android-settings-development-SpoofSignaturePreferenceController.java ] && (cp $PDIR/*-packages-apps-Settings-src-com-android-settings-development-SpoofSignaturePreferenceController.java ${SRC_TOP}/packages/apps/Settings/src/com/android/settings/development/SpoofSignaturePreferenceController.java || exit 3)
```

Comment:
could you pls use "if...then..fi" for this and the next check? the reason is that otherwise the script will break when the condition is not true. using the if..then..fi will ensure it only ends with false when there is a (real) problem